substrate/frame/examples/single-block-migrations/src/migrations/v1.rs

```diff
@@ -0,0 +1,223 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use frame_support::{
+	storage_alias,
+	traits::{Get, OnRuntimeUpgrade},
+};
+
+#[cfg(feature = "try-runtime")]
+use sp_std::vec::Vec;
+
+/// Collection of storage item formats from the previous storage version.
+///
+/// Required so we can read values in the old storage format during the migration.
+mod old {
```

Comment:
```suggestion
mod v0 {
```

I think it reads nicer in the code then like `v0::Value::<T>::...`.

Reply:
Good idea. Updated in the example and other usages in the codebase.